app/assets/javascripts/components/NewsletterSticky.js

```diff
@@ -53,6 +53,10 @@ define([
     this.closeButton = this.$el.find(this.config.closeClassSelector);
     $window = $(window);
 
+    var focusable = this.$el.find('a[href], area[href], input:not([disabled], [type=hidden]), select:not([disabled], [hidden]), textarea:not([disabled], [hidden]), button:not([disabled], [hidden]), iframe, object, embed, *[tabindex], *[contenteditable]');
```

Comment:
For the Ruby at least I think it's 120 characters. Couldn't find anything about it in https://github.com/moneyadviceservice/frontend-code-standards directly, however it refers to the AirBnB standard, which says 100 chars (https://github.com/airbnb/javascript#6.2)